README.md

```diff
@@ -1,6 +1,27 @@
 # Vtu11
 
-Vtu11 is a small C++ header-only library to write unstructured grids using the vtu file format. It keeps the mess of dealing with file writing in different formats away from you. Currently it does not add any features for setting up the required data structure because this vastly differs based on the context in which vtu11 is used.
+_Vtu11_ is a small C++ header-only library to write unstructured grids using the vtu file format. It keeps the mess of dealing with file writing in different formats away from you. Currently it does not add any features for setting up the required data structure because this vastly differs based on the context in which _vtu11_ is used.
+
+## How to include in your project
```

Comment:
I can add a section in a followup PR on how to include it as header only
Not everyone is using CMake so I think it makes sense to have both options

Reply:
Well if you don't use CMake, then the zlib part becomes a bit more involved I think. Because there won't be a configure header. Not sure if that even works. But I also thought about it. 

Probably the way to do this is though install? Then in the install folder you would have the configure header too.